development/wasm-filter-plugins.md

```diff
@@ -0,0 +1,162 @@
+# WASM Filter Plugins
+
+Fluent Bit currently supports integration of wasm plugins built as wasm/wasi objects for input and filter plugins only. The interface for the WASM filter plugins is currently under development but is functional.
+
+## Prerequisites
+
+### For Build fluent-bit
+
+For only execution of WASM input and filter plugins, there is no additional requirement.
+
+#### Building flb-wamrc (optional)
+
+For `flb-wamrc` support, users have to install llvm infrastructure, e.g:
```

Comment:
What is this? Can you introduce or link?

Reply:
I added a description for `flb-wamrc` as:

> `flb-wamrc` is just `flb-` prefixed AOT (Ahead Of Time) compiler that is provided from [wasm-micro-runtime](https://github.com/bytecodealliance/wasm-micro-runtime).